packages/cli/src/tokens/build/configs.ts

```diff
@@ -106,6 +107,43 @@ const colorModeVariables: GetStyleDictionaryConfig = ({ mode = 'light', theme },
   };
 };
 
+const colorCategoryVariables =
+  (category: ColorCategories): GetStyleDictionaryConfig =>
+  ({ mode, theme, [`${category}-color` as const]: color }, { outPath }) => {
+    const layer = `ds.theme.color`;
+    const selector = `[data-color="${color}"]`;
+
+    return {
+      usesDtcg,
+      preprocessors: ['tokens-studio'],
+      platforms: {
+        css: {
+          // custom
+          outPath,
+          mode,
+          theme,
+          selector,
+          layer,
+          //
+          prefix,
+          buildPath: `${outPath}/${theme}/`,
+          transforms: dsTransformers,
+          files: [
+            {
+              destination: `color/${color}.css`,
```

Comment:
Have you thought about combining all the colors into a single file, such as `colors.css`, which is a concat of builtin colors, support and main instead of having individual files for each color.

Reply:
It would make the generation here more complex for very little benefit, though. Right now one theme in $themes.json --> one css file is pretty easy to grasp.

Reply:
Fair enough